promql/engine.go

```diff
@@ -1494,6 +1495,9 @@ func (ev *evaluator) aggregation(op ItemType, grouping []string, without bool, p
 	var valueLabel string
 	if op == itemCountValues {
 		valueLabel = param.(string)
+		if !model.LabelName(valueLabel).IsValid() {
+			ev.errorf("Invalid Label Name '%s'", valueLabel)
```

Comment:
Taking another look, we use `%q` in the other parts of the code-base for quoting strings. Also, I think it should be all small letters, no capitals. Thoughts @brian-brazil?

Reply:
@juliusv was looking at that recently. They should probably all be lower case.

Reply:
@gouthamve I made the changes.